crates/annuity/src/lib.rs

```
@@ -108,7 +108,7 @@ pub mod pallet {
         pub fn withdraw_rewards(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
             let dest = ensure_signed(origin)?;
             let value = T::BlockRewardProvider::withdraw_reward(&dest)?;
-            let _ = T::Currency::transfer(&Self::account_id(), &dest, value, ExistenceRequirement::KeepAlive);
+            T::Currency::transfer(&Self::account_id(), &dest, value, ExistenceRequirement::KeepAlive)?;
```

Note this change, seems we discarded the error before but I'm not sure why